src/pages/Settings.tsx

```
@@ -94,27 +95,31 @@ export default function Settings() {
-                          const setting = category.settings.find((s) => s.id === field)!;
+                          const setting = category.settings.find((s) => s.id === field);
+                          if (!setting) return null;
                           return (
                             <div key={setting.id} className="space-y-2">
                               <Label htmlFor={setting.id} className="text-base font-medium">
                                 {setting.name}
                               </Label>
                               <Select
-                                value={getSetting(setting.id) as string}
-                                onValueChange={(value) => setSetting(setting.id, value)}
+                                value={getSetting(setting.id as any) as string}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)